cordova-lib/src/plugman/install.js

```diff
@@ -357,7 +356,7 @@ function runInstall(actions, platform, project_dir, plugin_dir, plugins_dir, opt
                     }
                 };
 
-                var hooksRunner = new HooksRunner(projectRoot);
+                var hooksRunner = new HooksRunner(cordovaUtil.isCordova() || project_dir);
 
```

Comment:
why `cordovaUtil.isCordova() || project_dir` ?
wouldn't just using `project_dir` achieve the same result ?